rpcs3/CMakeLists.txt

```diff
@@ -152,6 +152,11 @@ elseif(UNIX)
             COMMAND ${CMAKE_COMMAND} -E copy_directory
             ${CMAKE_SOURCE_DIR}/bin/git $<TARGET_FILE_DIR:rpcs3>/git)
 elseif(WIN32)
+    if(MSVC)
+        add_custom_command(TARGET rpcs3 POST_BUILD
+            COMMAND "${CMAKE_COMMAND}" -E copy "${PROJECT_BINARY_DIR}/3rdparty/OpenAL/openal-soft/$<CONFIG>/OpenAL32.dll" "$<TARGET_FILE_DIR:rpcs3>"
```

Comment:
Why not do this in the OpenAL build scripts?

Reply:
I provided it here due to it is the place where the application build is finalized (all the resource files are copied here (even the external qt files)